setup.cfg

```diff
@@ -58,9 +58,9 @@ dev =
 
 [flake8]
 max-line-length = 79
-# Ignored rules for Black
-# E203 whitespace before ':'
-ignore = E203
+# E203: whitespace before :, flake8 disagrees with PEP-8
+# W503: line break after binary operator, flake8 disagrees with PEP-8
+ignore = E203, W503
```

Comment:
This chunk turned out not to be necessary after refactoring, but it brings safir in line with the roundtable_aiohttp template so I left it.